global_helpers/gcp_environment.py

```diff
@@ -9,9 +9,9 @@
                     "system:serviceaccount:example-namespace:example-namespace-service-account"
                 ],
                 # If empty, then all namespaces
-                "namespaces": [],
+                "namespaces": ["example"],
```

Comment:
This was preventing the tests from working properly; I think this is a better test of the `all` logic which has three separate conditions that must be satisfied before the principal is verified.

Reply:
Better implementation: [`86e0b36` (#841)](https://github.com/panther-labs/panther-analysis/pull/841/commits/86e0b3650ecaa346373e31514a101bab661b486a)